### DISCUSSION TITLE
About surface pressure

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#728] [General]
Hello, I want to use surface pressure for flow field inversion. I have only found the experimental surface pressure curve, so I took points on the curve. But if I use the pData file, how can I put these points into this file? Or does using surface pressure for flow field inversion also require full field data?Looking forward to your reply.

**Comment by friedenhe** [comment]
You have to manually interpolate your experimental data to the ***boundaryField** of pData. No need to assign values for the internField in pData.
